### PR TITLE
LM-2827 allow to delete asset interests

### PR DESCRIPTION
**Pull request by waynesayonara**

### CODE REVIEW

File: src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs [hunk 1]
```diff
@@ -29,6 +29,19 @@ public async Task Update(AssetInterest assetInterest)
             await _dbContext.SaveChangesAsync();
         }
 
+        public async Task<int> DeleteByAssetIds(IReadOnlyCollection<string> assetIds)
+        {
+            var entities = await _dbContext.AssetInterests
```
Comment (KonstantinRyazantsev) [inline src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs]: Let's use Z.EntityFramework.Extensions bulk delete instead

Comment (waynesayonara) [reply src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs]: Unfortunately, both ways that I know of (`BulkDeleteAsync` and `DeleteFromQueryAsync`) result in the same error: `ERROR_005: The monthly trial period is expired`.

Comment (KonstantinRyazantsev) [reply src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs]: Check this https://github.com/swisschain/Sirius.Executor/blob/4e2fdbb8b99f8faf9f39d30d7baa7abc6527e96e/src/Executor.Common/Persistence/Entities/AddressLocks/AddressLocksRepository.cs#L80

Comment (waynesayonara) [reply src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs]: `+`

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**